boefjes/plugins/kat_burpsuite/normalize.py

```diff
@@ -0,0 +1,125 @@
+from xml.dom import minidom
```

Comment:
Since you're already using `bs4`, why not use that for XML parsing? It has a nicer API and using selectors will save some lines of code as well

Reply:
XML is a stricter parser then BS4 unless we configure bs4 to also use the xml parser that we now use directly.

Reply:
We could use the `lxml` parser backend, `bs4` plays very nicely with that one. It also supports schema validation if we really need that.